README.md

```diff
@@ -32,6 +32,8 @@ The earliest supported version is currently:
 
 ### Cookbook Dependencies
 
+None.
```

Comment:
Maybe we should take this section out entirely?

Reply:
see https://github.com/sous-chefs/postgresql/pull/561#discussion_r214646862